src/11.numberformat.js

```diff
@@ -104,10 +104,10 @@ export function /*11.1.1 */SetNumberFormatDigitOptions (intlObj, options, mnfdDe
     // 12. If mnsd is not undefined or mxsd is not undefined, then
     if (mnsd !== undefined || mxsd !== undefined) {
         // a. Let mnsd be ? GetNumberOption(options, "minimumSignificantDigits", 1, 21, 1).
-        mnsd = GetNumberOption(options, "minimumSignificantDigits", 1, 21, 1);
```

Comment:
this doesn't seem right. you're changing the actual algo step specified in 402. Can you explain why?

Reply:
I can't remember why. I agree that this should not change.